### PR TITLE
RXI-525 RPC parser unit tests

### PR DESCRIPTION
**Pull request by oleks-rip**

### CODE REVIEW

File: src/test/RPCParse_test.cpp [hunk 1]
```diff
@@ -0,0 +1,828 @@
...
+extern const char TxAccCreate[];
+extern const char TxCommit[];
+extern const char TxAccCreateAtt[];
+extern const char TxAccClaimAtt[];
+extern const char TxCreateBridge[];
+
+class RPCParse_test : public beast::unit_test::suite
+{
+public:
+    RPCParse_test()
```
Comment (seelabs) [inline src/test/RPCParse_test.cpp:41]: Could just `= default` this.

File: src/test/RPCParse_test.cpp [hunk 2]
```diff
@@ -0,0 +1,828 @@
...
+    {
+    }
+
+private:
+    void
+    testToString()
+    {
+        std::string s;
+        s = to_string(XChainTxnType::xChainCommit);
+        if (!BEAST_EXPECT(s == "xChainCommit"))
```
Comment (seelabs) [inline src/test/RPCParse_test.cpp:51]: I don't think we need the temp string or the early returns. So just:
```
        BEAST_EXPECT(to_string(XChainTxnType::xChainCommit) == "xChainCommit"));
```
for each of these is fine.

**Review by seelabs — APPROVED** [review]
:+1: looks good, but take a peek at the two comments.

File: src/test/RPCParse_test.cpp (hunk 2 above)
Comment (oleks-rip) [reply src/test/RPCParse_test.cpp]: fixed 245f34582c5

**Pull request merged**